client_tools.go

```diff
@@ -10,6 +10,7 @@ type IntKey struct {
 	I int
```

Comment:
Your changes should update golint-ci options.
Now all warnings suppressed with option ` --issues-exit-code=0 `:
```
      - name: golangci-lint
        uses: golangci/golangci-lint-action@v2
        with:
          args: --issues-exit-code=0 -E gofmt
```

Reply:
Fixed.